sunpy/physics/differential_rotation.py

```diff
@@ -42,10 +44,10 @@ def diff_rot(duration, latitude, rot_type='howard', frame_time='sidereal'):
 
     References
     ----------
-    | * `IDL code equivalent <http://hesperia.gsfc.nasa.gov/ssw/gen/idl/solar/diff_rot.pro>`_
-    | * `Howard rotation <http://adsabs.harvard.edu/abs/1990SoPh..130..295H>`_
-    | * `A review of rotation parameters (including Snodgrass values)
-    <http://link.springer.com/article/10.1023%2FA%3A1005226402796>`_
+
+    | * `IDL code equivalent <http://hesperia.gsfc.nasa.gov/ssw/gen/idl/solar/diff_rot.pro>`__
+    | * `Howard rotation <http://adsabs.harvard.edu/abs/1990SoPh..130..295H>`__
+    | * `A review of rotation parameters (including Snodgrass values) <http://link.springer.com/article/10.1023%2FA%3A1005226402796>`__
```

Comment:
Double underscore should generally be used for inline links, it basically means you can repeat the link elsewhere in the docs iirc.